src/test/testing_assert.cpp

```diff
@@ -94,88 +103,106 @@ ::testing::AssertionResult check_table_equal(const Table& tleft, const Table& tr
         right_col_type = "int";
       }
     }
-    if (left_col_type != right_col_type || tleft.column_name(col_id) != tright.column_name(col_id)) {
-      std::cout << "Column with ID " << col_id << " is different" << std::endl;
-      std::cout << "Got: " << tleft.column_name(col_id) << " (" << tleft.column_type(col_id) << ")" << std::endl;
-      std::cout << "Expected: " << tright.column_name(col_id) << " (" << tright.column_type(col_id) << ")" << std::endl;
-      print_tables();
-      return ::testing::AssertionFailure() << "Table schema is different.";
+    if (left_col_type != right_col_type || opossum_table->column_name(col_id) != expected_table->column_name(col_id)) {
+      std::cout << generate_table_comparison() << "Table schema is different."
```

Comment:
Do we want to print the tables even if the columns mismatch?

Reply:
always helps, imo